src/features/upload-file/providers/base-provider.ts

```diff
@@ -17,19 +17,25 @@ abstract class BaseProvider {
    * this type. We cannot check typeof because there could be a different versions of
    * npm package installed in a project.
    */
-  public name: string
+  public name: string;
 
   /**
    * Bucket is a place where files should be stored. Usually it is a folder location
    */
-  public bucket: string
+  public bucket: string;
+
+  /**
+   * Url is path prefix for local provider
+   */
+  public url: string;
```

Comment:
How about a different name which clearly indicates what it's for?
`baseUrl` or `urlPrefix` or `localUrl` or `uploadsUrl`

Reply:
changed